keyboards/neito/readme.md

```diff
@@ -0,0 +1,16 @@
+# Neito
+### by Oliver Granlund
+
+ISO-variant of TGR Alice with a few tweaks!
+
+![First prototype](https://i.imgur.com/GbZAnIpl.jpg)
+
+![First prototypes backside](https://i.imgur.com/h7Hx6EUl.jpg)
+
+GB:s of this board are hosted on [https://olli.works](https://olli.works)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make neito:default
+
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```

Comment:
Instructions on how to reset the board to bootloader mode should go above this line.